antora-playbook.yml

```diff
@@ -3,20 +3,20 @@ site:
   start_page: ROOT::overview.adoc
 content:
   sources:
-  # Test one component with each doc type being a module in it
   - url: https://github.com/OpenLiberty/docs.git
-    # start_path: .
-    branches: v20.0.0.7
+    branches: [v20.0.0.7, v20.0.0.8, v20.0.0.9, v20.0.0.10, v20.0.0.11, v20.0.0.12, v21.0.0.1, v21.0.0.2, v21.0.0.3, v21.0.0.4, v21.0.0.5, v21.0.0.6, v21.0.0.7, v21.0.0.8, v21.0.0.9, v21.0.0.10, v21.0.0.11, v21.0.0.12, v22.0.0.1, v22.0.0.2]
```

Comment:
I think your branch was based off prod, which will change the way the staging site will get built if we introduce all of these instead of the version-staging branches.